pkg/e2e/autoscaler/autoscaler.go

```diff
@@ -316,7 +316,6 @@ var _ = g.Describe("[Feature:Machines] Autoscaler should", func() {
 		initialNumberOfReplicas0 := pointer.Int32PtrDerefOr(targetMachineSet0.Spec.Replicas, e2e.DefaultMachineSetReplicas)
 		initialNumberOfReplicas1 := pointer.Int32PtrDerefOr(targetMachineSet1.Spec.Replicas, e2e.DefaultMachineSetReplicas)
 		glog.Infof("initialNumberOfReplicas0 %d, initialNumberOfReplicas1 %d", initialNumberOfReplicas0, initialNumberOfReplicas1)
-		o.Expect(initialNumberOfReplicas0).To(o.BeNumerically("==", initialNumberOfReplicas1))
```

Comment:
If we have two machinesets what prevents us from asserting this still holds true?

Reply:
One of the sets will have 2 replicas. The goal of openshift/release#3615 was to alleviate CI cloud rate limits by dropping one az (hence one machineSet) but keeping number of machines

Reply:
Ah, of course. My [current] PR has removed that assumption.